<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditor</class>
 <widget class="QDialog" name="ItemEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="questionLabel">
       <property name="text">
        <string>Question</string>
       </property>
       <property name="buddy">
        <cstring>questionLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="questionLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="answerLabel">
       <property name="text">
        <string>Answer</string>
       </property>
       <property name="buddy">
        <cstring>answerLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="answerLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="option1Label">
       <property name="text">
        <string>Option 1</string>
       </property>
       <property name="buddy">
        <cstring>option1LineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="option1LineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="option2Label">
       <property name="text">
        <string>Option 2</string>
       </property>
       <property name="buddy">
        <cstring>option2LineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="option2LineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="option4Label">
       <property name="text">
        <string>Option 3</string>
       </property>
       <property name="buddy">
        <cstring>option3LineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="option3LineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="categoryLabel">
       <property name="text">
        <string>Category</string>
       </property>
       <property name="buddy">
        <cstring>categoryComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="categoryComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="difficultyLabel">
       <property name="text">
        <string>Difficulty</string>
       </property>
       <property name="buddy">
        <cstring>difficultyComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="difficultyComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
